### PR TITLE
wiki aet dictionary update

### PR DESCRIPTION
**Pull request by RdkLwg**
AET Dictionary update

## Description
A few updates in the dictionary

## Motivation and Context
#383

## Screenshots (if appropriate)
n/a

## Upgrade notes (if appropriate)
n/a

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x ] My code follows the [code style](/CONTRIBUTING.md#coding-conventions) of this project.
- [ x] I have reviewed (and updated if needed) the documentation regarding this change

---
I hereby agree to the terms of the AET Contributor License Agreement.

### CODE REVIEW

File: documentation/src/main/wiki/Dictionary.md [hunk 1]
```diff
@@ -13,20 +22,15 @@ a set of system modules that are crucial for the whole system to work. The AET s
 implementations of jobs that can perform a particular task (e.g. collect screenshots, compare sources, validate a page against *W3C HTML5*).
 
 *AET Maven Plugin*  
-a default client application for the AET system that is used to trigger the execution of the *Test Suite*.
-
-*Amazon Web Services*
```
Comment (wiiitek) [inline documentation/src/main/wiki/Dictionary.md]: :+1:  :)

File: documentation/src/main/wiki/Dictionary.md [hunk 2]
```diff
@@ -63,19 +73,28 @@ a module responsible for filtering the collected data before performing comparis
 *Data Storage*  
 a database abstraction layer which contains versioned data (data grid).
 
+*Execute Java Script Modifier*  
+a modifier that allows to execute JavaScript code.
```
Comment (wiiitek) [inline documentation/src/main/wiki/Dictionary.md]: We could add:  _on tested page._ to this sentence, what do you think?

Comment (RdkLwg) [reply documentation/src/main/wiki/Dictionary.md]: Good idea!

Comment (RdkLwg) [reply documentation/src/main/wiki/Dictionary.md]: Done.

File: documentation/src/main/wiki/Dictionary.md [hunk 3]
```diff
@@ -13,20 +22,15 @@ a set of system modules that are crucial for the whole system to work. The AET s
 implementations of jobs that can perform a particular task (e.g. collect screenshots, compare sources, validate a page against *W3C HTML5*).
 
 *AET Maven Plugin*  
-a default client application for the AET system that is used to trigger the execution of the *Test Suite*.
-
-*Amazon Web Services*  
-cloud computing services where the AET environment can be deployed to.
+a default client application for the AET system that was used to trigger the execution of the *Test Suite*.
```
Comment (Skejven) [inline documentation/src/main/wiki/Dictionary.md]: Maven plugin is no longer default :)
It is just `a client application`.

Comment (RdkLwg) [reply documentation/src/main/wiki/Dictionary.md]: Good point! I missed that. done.

**Pull request merged**